jsapp/js/dataInterface.es6

```diff
@@ -258,10 +258,10 @@ var dataInterface;
     assignPerm (creds) {
       // Do we already have these URLs stored somewhere?
       var objectUrl = creds.objectUrl || `${ROOT_URL}/${creds.kind}s/${creds.uid}/`;
-      var userUrl = `${ROOT_URL}/users/${creds.username}/`;
+      var userUrl = `${ROOT_URL}/api/v2/users/${creds.username}/`;
       var codename = creds.role.includes('_submissions') ? creds.role : `${creds.role}_${creds.kind}`;
       return $ajax({
-        url: `${ROOT_URL}/permissions/`,
+        url: `${ROOT_URL}/api/v2/${assetUid}/permissions/`,
```

Comment:
I'm guessing that this also shouldn't have `${assetUid}/` part?

Reply:
@noliveleger @arsea I did all changes related to this, @arsea would you like to review the changes?

Reply:
I think @arsea might be too busy to look at this; I'll review it now